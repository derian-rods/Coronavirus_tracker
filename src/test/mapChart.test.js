import React from 'react';
import '@testing-library/jest-dom';
import MapChart from '../components/Map';
import { shallow } from 'enzyme';

describe('test for chart map component', () => {

    test('should show <Mapchart /> correctly', () => {
        const countries = [
            {
                country: 'Spain', 
                countryInfo:{
                    flag: 'https://disease.sh/assets/img/flags/af.png'
                }
            }, 
            {  country: 'EEUU', 
               countryInfo:{
                 flag: 'https://disease.sh/assets/img/flags/af.png'}
            },
        ]

        const wrapper = shallow(<MapChart countries={countries}/>)

        expect(wrapper).toMatchSnapshot()
    })
    

});