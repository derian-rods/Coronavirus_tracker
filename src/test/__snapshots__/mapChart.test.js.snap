// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for chart map component should show <Mapchart /> correctly 1`] = `
<div
  className="app__map"
>
  <div
    className="card__map"
  >
    <h2
      className="map__title"
    >
      World Map
    </h2>
    <div
      className="map__table"
    >
      <Table />
    </div>
    <ComposableMap
      projectionConfig={
        Object {
          "rotate": Array [
            -10,
            0,
            0,
          ],
          "scale": 160,
        }
      }
    >
      <ZoomableGroup>
        <Memo(Sphere)
          stroke="#E4E5E6"
          strokeWidth={0.5}
        />
        <Memo(Graticule)
          stroke="#E4E5E6"
          strokeWidth={0.5}
        />
      </ZoomableGroup>
    </ComposableMap>
  </div>
</div>
`;
